shared/core/types/proto/relay.proto

```diff
@@ -6,24 +6,37 @@ option go_package = "github.com/pokt-network/pocket/shared/core/types";
 
 // TECHDEBT: These structures were copied as placeholders from v0 and need to be updated to reflect changes in v1
 message Relay {
-    RelayPayload payload = 1;
-    RelayMeta meta = 2;
+    RelayMeta meta = 1;
+    oneof RelayPayload {
+        JSONRPCPayload json_rpc_payload = 2;
+        RESTPayload rest_payload = 3;
+        //DISCUSS: content of the following types:
+        // GRPCPayload grpc_payload = 3;
+        // GraphQLPayload graphql_payload = 4;
+        // WebSocketsPayload websockets_payload = 5;
+    }
 }
 
-message RelayPayload {
-    string data = 1;
-    string method = 2;
-    string http_path = 3;
-    map<string, string> headers = 4;
+message RESTPayload {
+    string contents = 1;
+}
+
+message JSONRPCPayload {
+    bytes id = 1;
+    string json_rpc = 2;
+    string method = 3;
+    bytes parameters = 4;
+    // DISCUSS: is HttpPath needed and is it a Meta field or Payload field?
```

Comment:
I don't really see/understand how `http_path` would be used in json-rpc because the method captures the functionality we are calling.

Can you:
1. Move http_path from `JSONRPCPayload` to `RESTPayload`
2. Add an enum in `RESTPayload` for the request type; `GET/SET/DELETE/etc`

Reply:
Thank you for the review. Fixed (As a side-note, on AVAX chain there is currently a path being used on portal, but that is more of a work-around, and a node can just set its URL to include the path instead)